### PR TITLE
Non-root cargo manifest file

### LINKED ISSUE #210: Rust: add support to non-root folder Config.toml
`cargo fmt` has support to `--manifest-path` argument, that can be used for passing `Config.toml` location.

It can be used in this way: `cargo fmt --manifest-path my/subdir/Cargo.toml -- --check`.

It is useful when your repository contains `Cargo.toml` in the non-root location.

### PR DESCRIPTION
**Pull request by AJIOB**
Should implement & fix #210

### CODE REVIEW

**Review by macisamuele — APPROVED** [review]
Change looks good.
If tests are green this will be merged and released shortly

**Comment by AJIOB** [comment]
I've fixed the tests passing & rebase to the latest master changes.

@macisamuele, recheck the PR, please

File: tests/pretty_format_rust_test.py [hunk 1]
```diff
@@ -30,16 +30,50 @@ def undecorate_method():
...
+        ("not-pretty-formatted_subdir/src/bin/src/main.rs", 1),
+    ),
+)
+def test_pretty_format_rust_manifest(undecorate_method, filename, expected_retval):
+    manifest_root = filename.split("/")[0]
+    manifest_root = os.path.abspath(manifest_root)
+    filename = os.path.abspath(filename)
+
+    manifest_file = os.path.join(manifest_root, "Cargo.toml")
+    print(manifest_file)
```
Comment (macisamuele) [inline tests/pretty_format_rust_test.py:62]: remove print s=from tests

**Review by macisamuele — APPROVED** [review]
Thanks for fixing the tests.
I would suggest to rebase this PR on top of latest `master`.

After that I should be able to merge this change.

**Comment by AJIOB** [comment]
All was fixed & updated. CI is also green.

**Pull request merged**